actors/common/src/main/java/org/sunbird/learner/actors/bulkupload/BulkUploadManagementActor.java

```diff
@@ -431,6 +435,7 @@ private void uploadCsvToDB(
     Response res =
         cassandraOperation.insertRecord(bulkDb.getKeySpace(), bulkDb.getTableName(), map);
     res.put(JsonKey.PROCESS_ID, processId);
+    ProjectLogger.log("BulkUploadManagementActor:uploadCsvToDB method end", LoggerEnum.INFO);
```

Comment:
From:
"BulkUploadManagementActor:uploadCsvToDB method end"

To:
"BulkUploadManagementActor: uploadCsvToDB send response"